Makefile

```diff
@@ -54,6 +54,9 @@ IMG ?= $(IMAGE_TAG_BASE):latest
 # Create Kubernetes objects with embeddedObjectMeta
 CRD_OPTIONS ?= "crd"
 
+WEBHOOK_IMG_BASE ?= k8ssandra/k8ssandra-secrets-webhook
+WEBHOOK_IMG ?= $(WEBHOOK_IMG_BASE):latest
```

Comment:
@adejanovski how will we handle versioning for this new image? Do we version it in lockstep with the existing k8ssandra-operator image? Do we want to publish a snapshot image on every commit?